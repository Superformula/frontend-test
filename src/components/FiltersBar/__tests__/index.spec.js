import React from 'react';
import { shallow } from 'enzyme';
import { FiltersBar } from '../';

jest.mock('hooks/useAction');
jest.mock('hooks/useSelector');

describe('FiltersBar', () => {
  it('should render without failing', () => {
    const wrapper = shallow(<FiltersBar />).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
