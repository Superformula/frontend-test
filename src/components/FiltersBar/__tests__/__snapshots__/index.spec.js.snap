// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FiltersBar should render without failing 1`] = `
.c0 {
  display: none;
}

@media (min-width:758px) {
  .c0 {
    display: block;
    position: -webkit-sticky;
    position: sticky;
    top: 0;
    background-color: #FFF;
    z-index: 1;
  }
}

<div
  className="c0"
>
  <Separator
    spaced={true}
  />
  <GridWrapper
    flex={true}
  >
    <Label>
      Filter by
      :
    </Label>
    <Filters />
    <ClearAll
      $fill={false}
      $secondary={false}
      disabled={true}
    >
      Clear all
    </ClearAll>
  </GridWrapper>
  <Separator
    spaced={true}
  />
</div>
`;
